QuestionBot/QuestionBot/Model/QuestionMessageListener.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace QuestionBot.Model{
+
+    public class QuestionMessageListener : IMessageListener{
+
+        private IStore _questionDataStore;
+        private const string QuestionCommand = "/question";
+
+        public QuestionMessageListener(IStore store){
+            _questionDataStore = store;
+        }
+
+        public string ReceiveMessage(string message){
+            string outputMessage;
+
+            if (String.IsNullOrEmpty(message) || !message.StartsWith(QuestionCommand)){
+                return null;
+            }
+
+            string questionText = message.Remove(0, QuestionCommand.Length);
+            questionText = questionText.TrimStart(' ');
```

Comment:
What if `message` was something like <code>"/question What is 1 + 1?&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"</code> Or what if it contained whitespace that isn't a space (ie tab)?